.github/workflows/build_push_img.yml

```diff
@@ -0,0 +1,132 @@
+name: Build and Push Marqo Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      marqo_ref:
+        description: 'Marqo branch-name, commit SHA or tag'
+        required: false
+        default: 'mainline'
+      push_to:
+        description: 'image destination. Options: "ECR", "DockerHub"'
+        required: true
+      image_repo:
+        description: 'Image repository'
+        required: true
+        default: 'marqo'
+      image_tag:
+        description: 'Image tag'
+        required: true
+      dockerhub_username:
+        description: 'DockerHub username'
+        required: false
+      dockerhub_password:
+        description: 'DockerHub password'
+        required: false
+
+jobs:
+  Start-Runner:
+    # on US-WEST-2, open source account
+    name: Start self-hosted EC2 runner
+    runs-on: ubuntu-latest
+    outputs:
+      label: ${{ steps.start-ec2-runner.outputs.label }}
+      ec2-instance-id: ${{ steps.start-ec2-runner.outputs.ec2-instance-id }}
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_EC2_GH_RUNNER_ACCESS_KEY_ID }}
```

Comment:
What is the AWS account? OpenSource or S2Search Core?

Reply:
This is open source now - I changed the secrets to use opensource